rules/network/command_and_control_port_26_activity.toml

```diff
@@ -36,8 +36,7 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-(event.dataset: network_traffic.flow or (event.category: (network or network_traffic))) and
-    network.transport:tcp and (destination.port:26 or (event.dataset:zeek.smtp and destination.port:26))
+(event.dataset: (network_traffic.flow or zeek.smtp) or event.category:(network or network_traffic)) and network.transport:tcp and destination.port:26
```

Comment:
The indices used for this rule are outside of the NPC integration, they are any beats related ingestion. My question is more around the `or` statement.

With this tuning, we are going to create alerts on ANY dport 26 traffic as long as it is TCP within filebeats. It may not even be SMTP, we assume so based on IANA's port mappings.

Reply:
Had a discussion with Mika about this rule. From the original logic, the scenario of any dport 26 traffic with any network data ingested through beats would still flag the same events. 

With this in mind, the new suggested logic is the same. However, I feel that this rule and scenario could be revisited at a later time to enhance the logic and scope of the threat.

Reply:
++ Its just to make sure we don't inadvertently have logic where one branch on the query would match `event.dataset: network_traffic.flow and event.dataset:zeek.smtp` which can never happen in a single event.